plugins/secretstores/docker/README.md

```diff
@@ -72,7 +72,12 @@ Referencing the secret within a plugin occurs by:
   password = "@{docker_secretstore:secret_for_plugin}"
 ```
 
-## Additonal Information
+To see which plugins and options support secrets, see their respective
+documentation (e.g. `plugins/outputs/influxdb/README.md`). If the plugin's
+README has the `Secret-store support` section, it will detail what options
+support secret store usage.
+
+## Additional Information
```

Comment:
Why is this text different than the ones below? Please see my comment below on including the text rather than copy-and-pasting it manually.

Reply:
There already was usage information in that README.md, and I didn't want to change that for the worse.